instance-api/order-api.md

```diff
@@ -0,0 +1,362 @@
+                            "owner": "true",
+                            "price": "12",
+                            "terms": "true",
+                            "title": "WOMM",
+                            "status": "released",
+                            "currency": "eur",
+                            "useCaseId": 11,
+                            "amountType": "unique",
+                            "redeemType": "physical",
+                            "description": "",
+                            "licenseType": "private",
+                            "nftToUpgrade": null,
+                            "secretPhrase": "",
+                            "storageOption": "s3",
+                            "fractionsAmount": "100000",
+                            "assetSameAsCover": true,
+                            "redeemItemsAmount": 1,
+                            "fractionableEnabled": true,
+                            "redeemValidationDate": null
+                        },
+                        "price": 1200,
+                        "priceEths": "13.701836617016",
+                        "priceCurrency": "EUR",
+                        "priceType": "FIXED",
+                        "downloadable": false,
+                        "title": "WOMM",
+                        "creatorId": 5,
+                        "useCaseId": 11,
+                        "upgradeable": false,
+                        "tokenId": null,
+                        "newTokenId": null,
+                        "oldTokenId": null,
+                        "status": "released",
+                        "releaseAt": "2022-12-09T11:22:17.543Z",
+                        "biddingEnabled": false,
+                        "createdAt": "2022-12-09T11:22:17.543Z",
+                        "updatedAt": "2022-12-09T12:55:09.001Z",
+                        "deletedAt": null,
+                        "fractionalized": true,
+                        "redeemable": false,
+                        "qrcodePrintfileId": null,
+                        "salesContractUrl": null
+                    },
+                    "listing": {
+                        "id": 4,
+                        "priceCents": 1200,
+                        "amount": 1,
+                        "priceEths": "13.651116149384164",
+                        "priceWei": "13651116149384164000",
+                        "activeTill": null,
+                        "active": true,
+                        "type": "regular",
+                        "txHash": null,
+                        "priceCurrency": "EUR",
+                        "sellerId": 5,
+                        "nftId": 406,
+                        "captureCronId": null,
+                        "createdAt": "2022-12-09T12:55:08.986Z",
+                        "updatedAt": "2022-12-09T12:55:08.986Z"
+                    }
+                }
+            ]
+        }
+    ],
+    "error": null
+}
+```
+
+## Obtaining the API Token
```

Comment:
Can you move this paragraph to a section: https://docs.license.rocks/authorization